src/forms/eventEditSchema.ts

```diff
@@ -0,0 +1,23 @@
+import * as Yup from "yup";
+
+import { EventInput } from "api/admin";
+
+export const eventEditSchema = Yup.object().shape<EventInput>({
+  name: Yup.string().required("Name required"),
+  description: Yup.string().required("Description required"),
+  start_date: Yup.string()
+    .required("Start date required")
+    .matches(
+      /\d{4}-\d{2}-\d{2}/,
```

Comment:
probably OSS, but I think it would be good to have some sort of a `config` file with all the regex/constants etc listed in

Reply:
there are already, `validationSettings.ts` (in this PR) and `dateSettings.ts` (from staging already) and more can be utilized as needed.

Reply:
This PR should help transition to more refactors later on, but for now the most important thing is to not cause too many merge conflicts (one PR for moving files, next one for moving RegExps, etc.)